lib/MetaCPAN/Server/Model/CPAN.pm

```diff
@@ -11,9 +11,9 @@ extends 'Catalyst::Model';
 with 'CatalystX::Component::Traits';
 
 has esx_model => (
-    is      => 'ro',
-    handles => ['es'],
-    default => sub { MetaCPAN::Model->new( es => $_[0]->servers ) },
+    is         => 'ro',
+    lazy_build => 1,
```

Comment:
this one is probably not relevant anymore. there are no mentions of
`lazy_build` in the code now.

On Mon, Apr 18, 2016 at 1:47 PM, Olaf Alders <notifications@github.com>
wrote:

> In lib/MetaCPAN/Server/Model/CPAN.pm
> <https://github.com/CPAN-API/cpan-api/pull/451#discussion_r60044267>:
>
> > @@ -11,9 +11,9 @@ extends 'Catalyst::Model';
> >  with 'CatalystX::Component::Traits';
> >
> >  has esx_model => (
> > -    is      => 'ro',
> > -    handles => ['es'],
> > -    default => sub { MetaCPAN::Model->new( es => $_[0]->servers ) },
> > +    is         => 'ro',
> > +    lazy_build => 1,
>
> This can still be lazy => 1 with a default, but let's merge this now and
> we can look at it later.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/CPAN-API/cpan-api/pull/451/files/bb2971e4774fc3c47e7531bdb7822921f68060d7..1a7a6897b42728a46e390e0536289644f648d268#r60044267>
>